tests/samplers_tests/test_nsgaii.py

```diff
@@ -902,7 +902,7 @@ def test_crossover_duplicated_param_values(crossover: BaseCrossover) -> None:
         (SBXCrossover(), 0.0, np.array([2.0, 3.0])),  # c1 = (p1 + p2) / 2.
         (SBXCrossover(), 0.5, np.array([3.0, 4.0])),  # p2.
         (SBXCrossover(), 1.0, np.array([3.0, 4.0])),  # p2.
-        (VSBXCrossover(), 0.0, np.array([2.0, 3.0])),  # c1 = (p1 + p2) / 2.
+        (VSBXCrossover(), 0.0, np.array([-inf, -inf])),  # c1 = (p1 + p2) / 2.
```

Comment:
Random is used in `beta_1`, the variable handled in the crossover calculation. If the value of this random is 0, the zero division occurs, so it is inf.
The random variable will be the one corresponding to `u` in the paper and `us` in the implementation. The range of `u` is [0,1] in the reference paper, and this problem occurs.

Reply:
I consider that avoiding division by zero is better in the implementation since it fails optimization.
```python
  File "/Users/yozaki/work/review/optuna/optuna/samplers/nsgaii/_crossovers/_vsbx.py", line 94, in crossover
    beta_1 = np.power(1 / (2 * us), 1 / (eta + 1))
                      ~~^~~~~~~~~~
ZeroDivisionError: division by zero
```